client-frameworks-support/client-support-ui5/src/ui5-support-lib.js

```diff
@@ -0,0 +1,107 @@
+sap.ui.define(['sap/ui/model/json/JSONModel', '@luigi-project/client/luigi-client'], function(JSONModel, LuigiClient) {
+  const model = new JSONModel();
+  LuigiClient.addInitListener(ctx => {
+    model.setData(ctx);
+  });
+
+  LuigiClient.addContextUpdateListener(ctx => {
+    model.setData(ctx);
+  });
+
+  return {
+    connectTo: function(oComponent) {
+      const routingConfig = oComponent.getManifestEntry('sap.ui5').routing;
+      oComponent.setModel(model, '$luigiCtx');
+
+      // Create preload view
+      sap.ui.define('luigi/DummyView', ['sap/ui/core/mvc/View', 'sap/ui/core/Control'], function(View) {
+        const DummyView = View.extend('luigi.DummyView', {
+          getControllerName: function() {},
+          createContent: function() {
+            return new sap.ui.core.HTML({ content: '<div></div>' });
```

Comment:
AHHH - Global - should be required in `sap.ui.define` above for the `LuigiView` instead of the `sap/ui/core/Control`, e.g.:

```
sap.ui.define('luigi/DummyView', ['sap/ui/core/mvc/View', 'sap/ui/core/HTML'], function(View, HTML) {
[...]
return new HTML({ content: '<div></div>' });
[...]
```